### DISCUSSION TITLE
Welcome to jhipster-dotnetcore Discussions!

### DISCUSSION THREAD

**Discussion by nicolas63** [jhipster/jhipster-dotnetcore#486] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ivanmonteiro** [comment]
Thank you @nicolas63 for starting this discussion. It will be great to share new ideas. 

> To get started, comment below with an introduction of yourself and tell us about what you do with this community.

Well, I have been programming with .Net for the last ~8 years. I have a bachelor's degree in Information Systems and I'm studying for a Masters degree in Computer Science at UFPE (Brazil).  

I have first known the main jhipster (java) generator in 2017 and I have a growing interest in the dotnet blueprint. I started contributing to this project last year and I have learned a lot.
